vanilla-framework/vanilla-framework
# 1544: Add simple search functionality to docs sidenav

## Done

- Added a search box to the sidenav in docs that filters the Vanilla patterns
- Updated header nav to match [vanillaframework.io](https://vanillaframework.io/)
- Updated docs to Vanilla 1.6.6 (will be released very soon)

## QA

- Pull code
- Run `cd docs && ./run`
- Open http://0.0.0.0:8104/
- Check that there is a search box in the sidebar navigation
- Check that it filters the results properly (it's just a simple title search, not a content search)

## Details

Fixes #1544
